packages/styles/src/variables/_commons.scss

```diff
@@ -30,26 +30,28 @@ $caret-width: 0.3em !default;
 // of components dependent on the z-axis and are designed to all work together.
 
 $zindex-spinner: 990 !default; // Design System only
-$zindex-dropdown: 1000 !default;
 $zindex-header: 1020 !default;
+$zindex-toast: 1080 !default;
+
+$zindex-map: (
+  n1: -1,
+  0: 0,
+  1: 1,
+  2: 2,
+  3: 3,
+);
+
+$named-zindex-map: (
+  'spinner': $zindex-spinner,
+  'header': $zindex-header,
+  'toast': $zindex-toast,
```

Comment:
We also need a `fixed` z-index: https://github.com/swisspost/design-system/pull/4863/files#diff-7963a80ea55e6328e7fe54f3156eb710db616b0e54e98bf200524add43075e2aR35